src/main/scala/com/ing/wbaa/rokku/proxy/provider/aws/SignatureHelpersCommon.scala

```diff
@@ -30,6 +30,7 @@ trait SignatureHelpersCommon {
     "%3A" -> ":",
     "%3C" -> "<",
     "%3E" -> ">",
+    "%3D" -> "=",
```

Comment:
should this stuff just be handled by a URL encoder? Or is it different from that?